app/controllers/propertysController.js

```diff
@@ -1,60 +1,69 @@
-import { propertys } from '../models/propertys';
+import { propertys, Property } from '../models/propertys';
+import dotenv from 'dotenv';
 
+dotenv.config();
 
 export class PropertyController {
-    postPropertyAdvert(req,res){ 
-        const property = {
-            id: propertys.length +1,
-            owner: req.body.owner ,
-            status: 'available',
-            price: req.body.price,
-            state: req.body.state,
-            city: req.body.city,
-            address: req.body.address,
-            type: req.body.type,
-            createdOn: Date.now(),
-            image : req.body.image
-        }
-    propertys.push(property);
-    res.status(201).send({message: 'success', property})
-};
+    postPropertyAdvert(req, res) {
+      const {owner, price, state, city, address, type, imageurl } = req.body;
+      const data = [owner, price, state, city, address, type,imageurl]
+     const newAdvert = Property.postPropertyAdvert(data);
+     newAdvert.then(ad => res.status(201).send({status: 201, message: 'success',data:ad.rows[0]}));
+    }
+
+
+
+//   postPropertyAdvert(req, res) {
+//     const property = {
+//       id: propertys.length + 1,
+//       owner: req.body.owner,
+//       status: 'available',
+//       price: req.body.price,
+//       state: req.body.state,
+//       city: req.body.city,
+//       address: req.body.address,
+//       type: req.body.type,
+//       createdOn: Date.now(),
+//       image: req.body.image,
+//     };
+//     propertys.push(property);
+//     res.status(201).send({ message: 'success', property });
+//   }
 
 // find (ad)property id then update details
-updateProperty(req,res){
-    const property = propertys.find(ad => ad.id === parseInt(req.params.id))
-    if(!property) return res.status(404).send({error:404, message:'property not found'})
+updateProperty(req, res) {
+    const property = propertys.find(ad => ad.id === parseInt(req.params.id));
+    if (!property) return res.status(404).send({ error: 404, message: 'property not found' });
     property.price = req.body.price,
     property.type = req.body.type,
-    res.status(201).send({message: 'success', property})
-    
-};
+    res.status(201).send({ message: 'success', property });
+  }
 // find (a)property id then mark as sold 
-markAsSold(req,res){
-    const property = propertys.find(a => a.id === parseInt(req.params.id))
-    if(!property) return res.status(404).send({error:404, message:'property of that id not found'})
+markAsSold(req, res) {
+    const property = propertys.find(a => a.id === parseInt(req.params.id));
+    if (!property) return res.status(404).send({ error: 404, message: 'property of that id not found' });
     property.status = 'sold',
-    res.status(201).send({message: 'success', property})
-};
+    res.status(201).send({ message: 'success', property });
+  }
 // find (d)property id
```

Comment:
what is the purpose of this comment? If it doesn't serve any purpose please remove it.